### PR TITLE
docs: update finetuner docs

### PR DESCRIPTION
**Pull request by jemmyshin**
since finetuner updates their model repo, we need to update our docs. Also we need to specify `to_onnx=True` in `finetuner.fit()`

### CODE REVIEW

File: docs/user-guides/finetuner.md [hunk 1]
```diff
@@ -174,10 +175,6 @@ executors:
     replicas: 1
 ```
 
-```{warning}
```
Comment (ZiniuYu) [inline docs/user-guides/finetuner.md:177]: Do not remove it yet. Change it to list 4 models not yet supported by Finetuner (and maybe M-CLIP)

**Review by ZiniuYu — CHANGES_REQUESTED** [review]

File: docs/user-guides/finetuner.md [hunk 2]
```diff
@@ -7,6 +7,8 @@ This guide will show you how to use [Finetuner](https://finetuner.jina.ai) to fi
 For installation and basic usage of Finetuner, please refer to [Finetuner documentation](https://finetuner.jina.ai).
 You can also [learn more details about fine-tuning CLIP](https://finetuner.jina.ai/tasks/text-to-image/).
 
+We use `finetuner`==0.6.2, `clip-as-service`==0.8.0 and `docarray`==0.17.0 in this tutorial.
```
Comment (numb3r3) [inline docs/user-guides/finetuner.md:10]:
```suggestion
We use `finetuner`==v0.6.2, `clip-as-service`==v0.8.0 and `docarray`==v0.17.0 in this tutorial.
```

Comment (numb3r3) [reply docs/user-guides/finetuner.md:10]: This tutorial only works on `clip-as-service==0.8.0` and `docarray=0.17.0`? I don't think so. This is misleading.

File: docs/user-guides/finetuner.md [hunk 3]
```diff
@@ -92,6 +94,7 @@ run = finetuner.fit(
     learning_rate=1e-5,
     loss='CLIPLoss',
     cpu=False,
+    to_onnx=True,
```
Comment (numb3r3) [inline docs/user-guides/finetuner.md:97]: As finetuner supports open_clip, can we finetune `model='ViT-B-32::openai'` in this tutorial.

File: docs/user-guides/finetuner.md [hunk 4]
```diff
@@ -174,10 +177,16 @@ executors:
...
 
+
 ```{warning}
-Note that Finetuner only support ViT-B/32 CLIP model currently. The model name should match the fine-tuned model, or you will get incorrect output.
+Note that `finetuner`==0.6.2 doesn't support these new clip models trained on Laion2B: 
+- ViT-B-32::laion2b-s34b-b79k
+- ViT-L-14::laion2b-s32b-b82k
+- ViT-H-14::laion2b-s32b-b79k
+- ViT-g-14::laion2b-s12b-b42k
 ```
```
Comment (numb3r3) [inline docs/user-guides/finetuner.md:187]: Can we provide the full model list supported in finetuner. This is not a release notes where the updated information is fine. As the official documentation, we should provide complete information.

**Review by numb3r3 — CHANGES_REQUESTED** [review]

File: docs/user-guides/finetuner.md [hunk 5]
```diff
@@ -92,6 +94,7 @@ run = finetuner.fit(
     learning_rate=1e-5,
     loss='CLIPLoss',
     cpu=False,
```
Comment (numb3r3) [inline docs/user-guides/finetuner.md:96]: BTW, I would suggest using the default `cpu` value in our doc.

**Review by numb3r3 — CHANGES_REQUESTED** [review]

File: docs/user-guides/finetuner.md (hunk 2 above)
Comment (ZiniuYu) [reply docs/user-guides/finetuner.md:10]: We can loose the restriction on docarray. This feature requires cas>=0.6.0. And it should be the `clip_server` package

File: docs/user-guides/finetuner.md (hunk 3 above)
Comment (ZiniuYu) [reply docs/user-guides/finetuner.md:97]: this model name does not match that in finetuner

File: docs/user-guides/finetuner.md (hunk 4 above)
Comment (jemmyshin) [reply docs/user-guides/finetuner.md:187]: Model names in finetuner are not the same as in cas, which is confusing. Should we creat a mapping for the model names?

File: docs/user-guides/finetuner.md [hunk 6]
```diff
@@ -7,6 +7,8 @@ This guide will show you how to use [Finetuner](https://finetuner.jina.ai) to fi
 For installation and basic usage of Finetuner, please refer to [Finetuner documentation](https://finetuner.jina.ai).
 You can also [learn more details about fine-tuning CLIP](https://finetuner.jina.ai/tasks/text-to-image/).
 
+This tutorial works under `finetuner`>=v0.6.2, `clip_server`>=v0.6.0 and `docarray`>=v0.17.0.
```
Comment (ZiniuYu) [inline docs/user-guides/finetuner.md:10]:
```suggestion
This tutorial requires `finetuner>=0.6.2` and `clip_server>=0.6.0`.
```

I suggest removing requirement for docarray and using pip install syntax

File: docs/user-guides/finetuner.md [hunk 7]
```diff
@@ -7,6 +7,8 @@ This guide will show you how to use [Finetuner](https://finetuner.jina.ai) to fi
 For installation and basic usage of Finetuner, please refer to [Finetuner documentation](https://finetuner.jina.ai).
 You can also [learn more details about fine-tuning CLIP](https://finetuner.jina.ai/tasks/text-to-image/).
 
+This tutorial requires `finetuner >=v0.6.2', `clip_server >=v0.6.0'.
```
Comment (bwanglzu) [inline docs/user-guides/finetuner.md:10]: let's stick with 0.6.3, we bumped GPU version

File: docs/user-guides/finetuner.md [hunk 8]
```diff
@@ -174,10 +176,16 @@ executors:
     replicas: 1
 ```
 
+
 ```{warning}
-Note that Finetuner only support ViT-B/32 CLIP model currently. The model name should match the fine-tuned model, or you will get incorrect output.
+Note that `finetuner`==0.6.2 doesn't support these new clip models trained on Laion2B:
```
Comment (bwanglzu) [inline docs/user-guides/finetuner.md:181]: 0.6.3

Comment (bwanglzu) [reply docs/user-guides/finetuner.md:181]: note, you can use `finetuner.describe_models()` to check the supported models

**Review by bwanglzu — COMMENTED** [review]
Thanks! Left some minor comments

File: docs/user-guides/finetuner.md [hunk 9]
```diff
@@ -174,10 +176,20 @@ executors:
...
 
+
 ```{warning}
-Note that Finetuner only support ViT-B/32 CLIP model currently. The model name should match the fine-tuned model, or you will get incorrect output.
+Note that `finetuner==0.6.3` doesn't support these new clip models trained on Laion2B: 
+- ViT-B-32::laion2b-s34b-b79k
+- ViT-L-14::laion2b-s32b-b82k
+- ViT-H-14::laion2b-s32b-b79k
+- ViT-g-14::laion2b-s12b-b42k
 ```
```
Comment (numb3r3) [inline docs/user-guides/finetuner.md:186]: remove this list

File: docs/user-guides/finetuner.md [hunk 10]
```diff
@@ -174,10 +176,20 @@ executors:
...
 
+```{tip}
+You can use finetuner.describe_models() to check the supported models.
```
Comment (numb3r3) [inline docs/user-guides/finetuner.md:189]: present the result

**Review by numb3r3 — CHANGES_REQUESTED** [review]

File: docs/user-guides/finetuner.md [hunk 11]
```diff
@@ -7,6 +7,8 @@ This guide will show you how to use [Finetuner](https://finetuner.jina.ai) to fi
 For installation and basic usage of Finetuner, please refer to [Finetuner documentation](https://finetuner.jina.ai).
 You can also [learn more details about fine-tuning CLIP](https://finetuner.jina.ai/tasks/text-to-image/).
 
+This tutorial requires `finetuner >=v0.6.3', `clip_server >=v0.6.0'.
```
Comment (ZiniuYu) [inline docs/user-guides/finetuner.md:10]: typo here

File: docs/user-guides/finetuner.md [hunk 12]
```diff
@@ -174,10 +176,54 @@ executors:
     replicas: 1
 ```
 
-```{warning}
-Note that Finetuner only support ViT-B/32 CLIP model currently. The model name should match the fine-tuned model, or you will get incorrect output.
+
+```{tip}
```
Comment (ZiniuYu) [inline docs/user-guides/finetuner.md:180]: Check the rendering of this table. Also, do we still need this?

File: docs/user-guides/finetuner.md [hunk 13]
```diff
@@ -91,7 +93,7 @@ run = finetuner.fit(
     epochs=5,
     learning_rate=1e-5,
     loss='CLIPLoss',
-    cpu=False,
+    to_onnx=True,
```
Comment (ZiniuYu) [inline docs/user-guides/finetuner.md:96]: Please change the model name to `ViT-B-32::openai`

File: docs/user-guides/finetuner.md [hunk 14]
```diff
@@ -174,10 +176,54 @@ executors:
     replicas: 1
 ```
```
Comment (ZiniuYu) [inline docs/user-guides/finetuner.md:177]: Please change the name to `ViT-B-32` or something else to match the latest CAS

**Review by ZiniuYu — CHANGES_REQUESTED** [review]

File: docs/user-guides/finetuner.md (hunk 11 above)
Comment (numb3r3) [reply docs/user-guides/finetuner.md:10]: `finetuner >= v0.6.4`

**Review by numb3r3 — CHANGES_REQUESTED** [review]

File: docs/user-guides/finetuner.md [hunk 15]
```diff
@@ -7,7 +7,7 @@ This guide will show you how to use [Finetuner](https://finetuner.jina.ai) to fi
 For installation and basic usage of Finetuner, please refer to [Finetuner documentation](https://finetuner.jina.ai).
 You can also [learn more details about fine-tuning CLIP](https://finetuner.jina.ai/tasks/text-to-image/).
 
-This tutorial requires `finetuner >=v0.6.3', `clip_server >=v0.6.0'.
+This tutorial requires `finetuner >=v0.6.4', `clip_server >=v0.6.0'.
```
Comment (ZiniuYu) [inline docs/user-guides/finetuner.md]: quotation mark typo :smile-wechat: please check rendering

**Review by ZiniuYu — CHANGES_REQUESTED** [review]

**Review by numb3r3 — APPROVED** [review]
LGTM

**Pull request merged**